scripts/webpack/webpack-resources.js

```diff
@@ -198,10 +189,8 @@ module.exports = {
         plugins: [
           // TODO: will investigate why this doesn't work on mac
           // new WebpackNotifierPlugin(),
-          new ForkTsCheckerWebpackPlugin()
-          // This sends output to stderr for some reason, which makes rush build say
-          // "succeeded with warnings" when there were no real warnings
-          // ...(process.env.TF_BUILD ? [] : [new webpack.ProgressPlugin()])
```

Comment:
Instead of removing this you could bring back the progress plugin now that it won't make rush assume the build is failing